TesSense.py

```diff
@@ -14,18 +19,17 @@
 """
 
 import datetime
-from time import sleep
-
-rate = minrate = 0                                    # Minimum rate you want to set the charger to
-volts = 120                                           # Minimum volts, until detected by the charger
-pshown = loop = charge = charging = False             # init more variables
-
 def printmsg(msg) :                                   # Timestamped message
     print( datetime.datetime.now().strftime( "%a %I:%M %p" ), msg )
     
 def printerror(error, err) :                          # Error message with truncated data
     print( datetime.datetime.now().strftime( "%a %I:%M %p" ), error+"\n", str(err).split("}")[0], "}" )
     
+#/c Set stdout as logging handler
```

Comment:
Not required, but since SenseLink uses logging, this causes it to spit those log calls out to stdout